conans/client/cmd/create.py

```diff
@@ -28,7 +28,7 @@ def _get_test_conanfile_path(tf, conanfile_path):
 
 def create(app, ref, graph_info, remotes, update, build_modes,
            manifest_folder, manifest_verify, manifest_interactive, keep_build, test_build_folder,
-           test_folder, conanfile_path, recorder, is_build_require=False):
```

Comment:
Does it also overrides build_requirements ?

Reply:
Only normal requires in this proposal. ``build_requires`` are not overriden in the same way, they don't have the same ``override`` concept. When you add a ``[build_requires]`` in your profile, it will inject that build_requires to dependencies, always, not just overriding the existing versions if they exist. It might be possible to overwrite the ``profile`` build_requires on the fly as well, but it should be clear that the behavior is quite different to this feature.